backend/README.md

```diff
@@ -55,6 +55,31 @@ $ yarn run start:prod
 ### On kubernetes
 Follow https://github.com/Substancia/REACTive-cucumber-NEST/blob/main/kubernetes/README.md
 
+### Setting up database 
+Step 1) For setting up the MongoDB instance in a docker container. We are skipping the steps involved in installing docker. 
+
+```bash
+# creating a dedicated folder in the host for storage of mongo db data 
+$ mkdir -p /data/db 
+
+# start a container by attaching storage to the folder in host machine 
+$ docker run -d -p 27017:27017 --name mongo-REACTive-cucumber-NEST -v data-vol:/data/db mongo:latest
```

Comment:
Can we rename the container to `cucumber-mongo` or `cucumber-database` for the sake of simplicity, yet understandability, as well as uniformity? We can name other containers as `cucumber-frontend` and `cucumber-backend`. Thoughts?

Reply:
If there's a bunch of other containers also running, on the same machine, `cucumber-` specific naming would be useful to identify. Let's do this.